source/documentation/using_the_api/getting_updates.md

```diff
@@ -1,42 +1,111 @@
-## Updating registers
+## Getting updates
 
-You can find the latest entry number by looking at the register information (use the [`GET /register` endpoint](#get-register)) and comparing the most recent entry number with your own copy.
+This guide assumes you know the current highest entry number for your copy of a register. 
 
-Example request: `https://local-authority-eng.register.gov.uk/register`
+You can regularly compare this value with new queries to a register to see if there are any updates.
 
-Download a full new copy of the register by using [the `GET /download-register` endpoint](#get-download-register).
+### Return all new entries since your last update
 
-Download all updates for one record according to a particular key by [using the `GET /records/{key}/entries` endpoint](#get-records-key-entries):
+You can use the [`GET /entries/` endpoint](#getentries) with the `start` and
+`limit` parameters to return specific entries to a register. 
```

Comment:
Is it possible to word it in a way such that the start/limit parameters are not encouraged to be used at will? And instead we favour the use  of the `Link` header?

Reply:
One way to do that is to not document them at all. We could just explain how to paginate using the headers.